project1/queries.py

```diff
@@ -0,0 +1,121 @@
+queries = ["" for i in range(0, 11)]
+### EXAMPLE
+### 0. List all airport codes and their cities. Order by the city name in the increasing order.
+### Output column order: airportid, city
+
+queries[0] = """
+select airportid, city
+from airports
+order by city;
+"""
+
+### 1. Write a query to find the names of customers who have flights on a friday and 
+###    first name that has a second letter is not a vowel.
+###    Do not include the youngest customer that satisfies the above conditions in the results.
```

Comment:
"Do not include the youngest customer **among those** that satisfies..."

Reply:
Done.